src/components/tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -162,6 +163,21 @@ describe('components/tooltip/Tooltip', () => {
             expect(wrapper.prop('targetAttachment')).toEqual('middle right');
         });
 
+        test('should render correct attachments when custom position is specified', () => {
+            const customPosition = {
+                attachment: TetherPosition.TOP_LEFT,
+                targetAttachment: TetherPosition.BOTTOM_RIGHT,
+            };
+            const wrapper = shallow(
+                <Tooltip position={customPosition} text="hi">
+                    <button />
+                </Tooltip>,
+            );
+
+            expect(wrapper.prop('attachment')).toEqual('top left');
+            expect(wrapper.prop('targetAttachment')).toEqual('bottom right');
```

Comment:
Can we use the constants from `TetherPosition` here as well?

Reply:
hi, I look at it again, since it is in the test, we can retain the hard coded string. see
https://medium.com/@erik.sacre/on-tdd-literals-versus-constants-in-tests-8ac992c7873f